### DISCUSSION TITLE
`rootDir` purpose w.r.to monaco editor TypeScript support

### DISCUSSION THREAD

**Discussion by manikantag** [microsoft/monaco-editor#3903] [Q&A]
Hi,

I'm not sure what is the purpose of [CompilerOptions.rootDir](https://microsoft.github.io/monaco-editor/docs.html#interfaces/languages.typescript.CompilerOptions.html#rootDir) w.r.to monaco editor usage.? I'm not seeing any difference when `rootDir` option is given or not given.

Is it needed when `noEmit = false` ?

Thanks

**Comment by tamayika** [comment]
This is just copied from typescript compiler option.
There are many unnecessary options for monaco-editor.
